src/types.ts

```diff
@@ -546,6 +560,22 @@ export interface MessageGetAddress extends MessageDefault {
   action: "getAddress";
 }
 
+// Liquid
+export interface MessageSignPsetWithPrompt extends MessageDefault {
+  args: {
+    pset: string;
+  };
+  action: "signPsetWithPrompt";
+}
+
+export interface MessageSignPset extends MessageDefault {
+  args: {
+    pset: string;
+    network: "liquid" | "testnet" | "regtest";
```

Comment:
can we use `LiquidNetworkType` here?

Reply:
@louisinger could you link me to the line of code in the source? how do I import it?

Reply:
sorry, my bad, we removed that type some time ago

Reply:
I guess it doesn't matter if we have another type with the same values. Updated in https://github.com/getAlby/lightning-browser-extension/pull/2583